pybio/spec/fields.py

```diff
@@ -0,0 +1,184 @@
+from importlib import import_module
+from urllib.parse import urlparse, ParseResult
+import json
+import pathlib
+import requests
+import subprocess
+import typing
+import yaml
+
+from marshmallow import ValidationError
+from marshmallow.fields import *
```

Comment:
Can we be explicit and import the names we need? It's easier to read then.

Reply:
changed. (I wanted fields to be an extension of marshmallow.fields such that in schema.py fields.AnyField can be used).